source/partials/drupal/dependencies-compatible.md

```diff
@@ -0,0 +1,29 @@
+---
+contenttype: [partial]
+categories: [--]
+newcms: [drupal, drupal8, drupal9]
+product: [--]
+integration: [--]
+tags: [--]
+reviewed: "2022-11-04"
+---
+
+You must confirm that your site is compatible with the latest version of Drupal before you change the Drupal core version.
+ 
+Review the steps below to confirm compatibility for your site's components.
+
+1. Optional. [Convert your site to Composer](/guides/composer-convert) if you plan to use the [Upgrade Status module](https://www.drupal.org/project/upgrade_status/) to assist in your upgrade.
+
+1. Verify that your hosting environment meets Drupal platform requirements.
+
+1. Update to Drupal 8.x-3.15 or higher.
+
+1. Update all contributed projects and verify that they are compatible with the latest version of Drupal.
```

Comment:
Maybe recommend installing and checking `upgrade_status` module first. https://www.drupal.org/project/upgrade_status.

Reply:
https://github.com/pantheon-systems/documentation/pull/8004/files#diff-87eb06ca232b52792135032479f07a19fe259fdaabe9fa76d1b5ab58392c804fR13 is about upgade_status

Reply:
added `1. Use [Upgrade Status](https://www.drupal.org/project/upgrade_status) to check the compatibility of all contributed modules and themes.` to top of list